src/modules/endpoint/models/CategoryModel.js

```diff
@@ -13,10 +13,10 @@ class CategoryModel {
   _thumbnail: string
   _order: number
   _availableLanguages: Map<string, string>
-  _lastUpdate: Moment
+  _lastUpdate: ?Moment
```

Comment:
Why is this optional?

Reply:
it actually was already Moment or null. Just fixed flow issues.